ext/Shared/__init__.lua

```diff
@@ -263,124 +51,129 @@ local function GetCustomLevel(p_LevelName, p_GameModeName)
 	return s_Preset
 end
 
+Hooks:Install('ResourceManager:LoadBundles', 100, function(p_Hook, p_Bundles, p_Compartment)
+	print(p_Compartment)
+	print(p_Bundles[1])
+end)
```

Comment:
Can be removed. Was just for debug prints.

Reply:
Yeah, but left it for the same purpose. The print is disabled so its fine,